interfaces/apparmor/template.go

```diff
@@ -453,10 +395,9 @@ var templateCommon = `
   /dev/ptmx rw,
 
   # Do the same with /sys/devices and /sys/class to help people using hw-assign
-  /sys/devices/ r,
-  /sys/devices/**/ r,
-  /sys/class/ r,
+  /sys/{class,devices}/ r,
   /sys/class/**/ r,
+  # /sys/devices/**/ r, # already enabled
```

Comment:
where is `/sys/devices/**/ r` already enabled?

Reply:
it is covered by line
`/sys/devices/**/{,read_ahead_kb,uevent} r,`